packages/suite/src/components/suite/PinMatrix/index.tsx

```diff
@@ -0,0 +1,171 @@
+import React, { useState, useEffect } from 'react';
+import styled, { css } from 'styled-components';
+import { DeviceImage, Icon, variables } from '@trezor/components';
+import { PinInput, Translation, TrezorLink } from '@suite-components';
+import { TrezorDevice } from '@suite-types';
+
+import { URLS } from '@suite-constants';
+import * as modalActions from '@suite-actions/modalActions';
+import { useActions, useTheme } from '@suite-hooks';
+
+const { FONT_SIZE, FONT_WEIGHT, SCREEN_SIZE } = variables;
+
+const Wrapper = styled.div`
+    display: flex;
+    flex-direction: row;
+    flex: 1;
+
+    @media only screen and (max-width: ${SCREEN_SIZE.MD}) {
+        flex-direction: column;
+    }
+`;
+
+const Col = styled.div<{ blur?: boolean }>`
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    border-radius: 5px;
+    width: 100%;
+    max-width: 340px;
+
+    ${props =>
+        props.blur &&
+        css`
+            filter: blur(5px);
+            pointer-events: none;
+        `}
+`;
+
+const GreyCol = styled(Col)`
+    background: ${props => props.theme.BG_GREY};
+    padding: 20px 24px;
+    margin-right: 34px;
+    max-width: 360px;
+
+    @media only screen and (max-width: ${SCREEN_SIZE.MD}) {
+        display: none;
+    }
+`;
+
+const Item = styled.div`
+    display: flex;
+    align-items: center;
+    width: 100%;
+    flex: 1;
+`;
+
+// const Divider = styled.div`
+//     width: 100%;
+//     height: 1px;
+//     background: ${props => props.theme.STROKE_GREY_ALT};
+// `;
+
+const ItemIconWrapper = styled.div`
+    display: flex;
+    width: 30px;
+    margin-right: 20px;
+    justify-content: center;
+`;
+
+const ItemText = styled.div`
+    width: 100%;
+    color: ${props => props.theme.TYPE_DARK_GREY};
+    font-size: ${FONT_SIZE.SMALL};
+    font-weight: ${FONT_WEIGHT.MEDIUM};
+    padding: 26px 0px;
+    text-align: left;
+    /* border-bottom: 1px solid #ccc; */
+`;
+
+const ExplanationCol = ({ invalid }: Pick<Props, 'invalid'>) => {
+    // TODO: Borders to match design
```

Comment:
Is there a follow up task for this? Or if its not needed anymore it, I guess the commented code above could be deleted too.

Reply:
I will add it to an issue with onboarding leftovers :)
https://github.com/trezor/trezor-suite/issues/3707

it is just a note I made a shortcut while implementing designs because it was not so straightforward and I didi't wanna waste more time. If it was up to me I would just delete it and consider it done, but let's see how product guys will react first 😁